frontend/src/js/components/tenants/tenantlist.tsx

```diff
@@ -75,7 +70,7 @@ const columnHeaders: ColumnHeader<Tenant>[] = [
       scope: ''
     },
     sortable: false,
-    textRender: DeviceLimitRender
+    component: DeviceLimitRender
```

Comment:
Could you please change line 39 to 
`  return (
    <div>
      {attributeValue}/{deviceLimit}
    </div>
  );
`